src/connection/directConnectionProvider.js

```diff
@@ -11,6 +11,9 @@ var AdmZip = require('adm-zip');
 var tar = require('tar');
 
 var ConnectionProvider = require('./../interface/connectionProvider');
+var LatestDriverVersionProvider = require('./latestDriverVersionProvider');
```

Comment:
Let's not use the Provider suffix as will conflict with ConnectionProvider interface. May use 'Resolver' for example.